synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Question
+about: If you have a question, please use Discussions
+
```

Comment:
I see that most of the others were changed from markdown -> yaml - should this one be also?

Reply:
Nope! It doesn't have any input fields.

Reply:
I could see that, but there might also be some value in just making them consistent for later edits.

Reply:
I'm going to keep it as is to discourage api-specific questions, see: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#body-must-contain-at-least-one-non-markdown-field